<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_window</class>
 <widget class="QWidget" name="menu_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>20</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>    菜单</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_multiplication">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>乘法练习</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_addition">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加法练习</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_back_main">
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>返回主页面</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_back_main</sender>
   <signal>clicked()</signal>
   <receiver>menu_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_addition</sender>
   <signal>clicked()</signal>
   <receiver>menu_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_multiplication</sender>
   <signal>clicked()</signal>
   <receiver>menu_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
